index.js

```diff
@@ -168,4 +173,30 @@ module.exports = {
     'wrap-iife': ERROR,
     yoda: [WARN, 'never'],
   },
+  overrides: [
+    {
+      files: ['*.ts'],
+      rules: {
+        '@typescript-eslint/array-type': 'warn',
+        '@typescript-eslint/no-var-requires': 'off',
+        '@typescript-eslint/no-use-before-define': 'off',
+        '@typescript-eslint/camelcase': 'off',
+        '@typescript-eslint/no-explicit-any': 'off',
+        '@typescript-eslint/interface-name-prefix': 'off',
+        '@typescript-eslint/explicit-function-return-type': 'off',
```

Comment:
Shouldn't this be set to 'on' ?

Reply:
Would you prefer? Why?

Reply:
It seems to me important to fully know what we will use, we should always indicate what a function should return, in order for the user to not be surprised by unwanted types.

Reply:
Thanks, done 🙂